### DISCUSSION TITLE
Logo Lights and Catering Truck

### DISCUSSION THREAD

**Discussion by longlive737** [flybywiresim/aircraft#2424] [Help]
I could not find an open discrepancy for these items so I wanted to check here before submitting a bug. I’m running the current developer build and a livery downloaded from flightsim.to. I am not sure if this is an issue with the base model, the developer build, or my own setup although I haven’t made .cfg changes which leads me to believe it’s one of the former two.

1. The logo light is not illuminating at all, even under the circumstances it should (gear compressed or slats extended). Nav lights illuminate normally.

2. When requesting catering service, the catering truck drives to the forward right door (#2) instead of door #4. Door #2 does not have an open animation, otherwise I wouldn’t mind.

If anyone else is also experiencing this that would make me feel less crazy. Thank you!

**Comment by wpine215** [comment]
Can you try running the mod under the following conditions:
- Default or FBW livery only (some liveries can cause conflicts)
- Temporarily remove all other mods in the community folder (other plugins/mods can cause conflicts)
- Try spawning at different airports/gates, both hand-crafted and non hand-crafted. Any issue with ground services is often tied to a particular airport/gate and has nothing to do with the mod. We are not able to change any of the ground service handling at all in our code.

**Reply by longlive737** [reply]
Hi wpine215,
So after simply re-installing the livery, logo lights work fine and correctly now. Don't know what happened there.

I completely uninstalled the FBW mod and tested the catering truck at several airports, to the same effect. This confirms what you said in that none of the FBW code affects ground services, and this issue is an Asobo one. 

Thanks for the help!
